addons/website_event/models/event_event.py

```diff
@@ -42,8 +42,17 @@ def _default_question_ids(self):
     # description
     subtitle = fields.Char('Event Subtitle', translate=True)
     # registration
-    is_participating = fields.Boolean("Is Participating", compute="_compute_is_participating")
+    is_participating = fields.Boolean("Is Participating", compute="_compute_is_participating",
+                                      search="_search_is_participating")
     # website
+    can_see_on_website = fields.Boolean(compute='_compute_can_see_on_website', search='_search_can_see_on_website')
+    full_website_url = fields.Char('Full Website Url', compute='_compute_full_website_url',
+                                   help='The full URL to the event')
+    visibility = fields.Selection(
+        [('public', 'Public'), ('link', 'Via a Link'), ('logged_users', 'Logged Users')],
+        required=True, default='public', tracking=True,
+        help="""Defines the Visibility of the Event on the Website and searches.\n
+            Note that the Event is however always available via its link.""")
```

Comment:
Meh, that's still confusing to me to say that if you choose "via a link", I fear people may think it's some kind of security which it's not.

But I've stated my opinion many times now, time to let it go 😄